MidCamp/midcamp
# Use drupal/core-dev for dev dependencies

## Description

Replaces independently called out Composer dev dependencies with `drupal/core-dev` for easier updating with Drupal Core.

## To test

Run the following commands to confirm preexisting packages are still in place:

- `composer show behat/mink`
- `composer show behat/mink-goutte-driver`
- `composer show composer/composer`
- `composer show drupal/coder`
